### DISCUSSION TITLE
Lost chia coins

### DISCUSSION THREAD

**Discussion by Sheva9005** [Chia-Network/chia-blockchain#12582] [Support]
Hello. I have a problem here. I’ve sent XCH from KuCoin account to my CHIA wallet, what I did before successfully. This time it lost, never received. When I open xchscan I got only 404 error message. Could you help me with this please? Thanks

**Comment by loppefaaret** [comment]
can you access the transaction details from kucoin ? it would be needed to assist in any way

**Comment by Sheva9005** [comment]
Good morning,
Next day I got my coins. Thank you for respond. Have a good day!

On Wed, Jul 27, 2022 at 4:02 AM loppefaaret ***@***.***>
wrote:

> can you access the transaction details from kucoin ? it would be needed to
> assist in any way
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Chia-Network/chia-blockchain/discussions/12582#discussioncomment-3258625>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A2GWTDQWHJSCKS5LFDUJ4SDVWDUIHANCNFSM54QSA7CQ>
> .
> You are receiving this because you authored the thread.Message ID:
> <Chia-Network/chia-blockchain/repo-discussions/12582/comments/3258625@
> github.com>
>
-- 
Best Regards
Andrey Shevchuk
+1 (305) 972-44-97
